src/partials/toc.hbs

```diff
@@ -4,9 +4,8 @@
 </div>
 {{> github}}
 {{> survey iconName="arrow-down"}}
-<h1 class="toc-title">On this page:</h1>
-<div class="aside-toc">
-  <nav class="toc-menu" aria-label="Skip to page content sections"></nav>
+<h1 class="toc-title">{{#if (eq (site-profile) 'jp' )}}このページの内容{{else}}On this page{{/if}}:</h1><div class="aside-toc">
```

Comment:
please move `<div class="aside-toc">` to its own line. Other than that LGTM

Reply:
@agansop I saw that you removed that line altogether. It is needed, just need to be put in a new line to make it easier to read.

Reply:
@cheungaryk should be fixed now